browser/ui/webui/new_tab_page/brave_new_tab_message_handler.cc

```diff
@@ -587,6 +591,20 @@ void BraveNewTabMessageHandler::HandleTodayOnCardVisits(
   int answer = it_count - kBuckets;
   UMA_HISTOGRAM_EXACT_LINEAR("Brave.Today.WeeklyMaxCardVisitsCount", answer,
                              base::size(kBuckets) + 1);
+  // Record ad click if a promoted card was read.
+  if (args->GetSize() < 4) {
+    return;
+  }
+  std::string item_id = args->GetList()[1].GetString();
+  std::string creative_instance_id = args->GetList()[2].GetString();
+  bool is_promoted = args->GetList()[3].GetBool();
+  if (is_promoted && !item_id.empty() && !creative_instance_id.empty()) {
+    auto* ads_service_ = brave_ads::AdsServiceFactory::GetForProfile(profile_);
+    ads_service_->OnBraveTodayAdEvent(
```

Comment:
Changed to `OnPromotedContentAdEvent` in latest ads library changes

Reply:
Changed